{
  "comments": [
    {
      "key": {
        "uuid": "37429346_ab8375e7",
        "filename": "/COMMIT_MSG",
        "patchSetId": 2
      },
      "lineNbr": 10,
      "author": {
        "id": 1208973
      },
      "writtenOn": "2018-03-19T22:03:07Z",
      "side": 1,
      "message": "Did you try deleting a label and making sure this does fail to build in that case?",
      "range": {
        "startLine": 10,
        "startChar": 0,
        "endLine": 10,
        "endChar": 18
      },
      "revId": "0522d760fdc43f52e0efcd0e90b3b44ee58081d4",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "5153fd58_9ce88e01",
        "filename": "tests/sepolicy_tests.py",
        "patchSetId": 2
      },
      "lineNbr": 15,
      "author": {
        "id": 1054468
      },
      "writtenOn": "2018-03-19T22:04:16Z",
      "side": 1,
      "message": "This doesn\u0027t (currently) work because it\u0027s based on file_contexts which is not used at all for \"proc\". Let me find my change which adds support for genfs_contexts here.",
      "range": {
        "startLine": 15,
        "startChar": 42,
        "endLine": 15,
        "endChar": 46
      },
      "revId": "0522d760fdc43f52e0efcd0e90b3b44ee58081d4",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": true
    }
  ]
}